### ISSUE TITLE
Lack of error description [Automatic download limit]

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1762]
Found by `HAN scan 08-24`:

In the ‘automatic download limit’ setting, if letters are entered instead of numbers, a status message containing the text ‘error’ appears on the screen. This message lacks any descriptive text, providing no information about the nature of the error or how to correct it. To resolve this, the error message should be more descriptive, clearly indicating the issue (e.g., ‘Please enter a number between 1-100’) to help users correct their input.
